script.module.libMediathek2/resources/language/English/strings.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
```

Comment:
would be cool if you would convert the xml language files to .po format.
xml format is deprecated.